CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/base/BaseActivity.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.coroutine.ui.base
+
+import android.os.Bundle
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+
+abstract class BaseActivity : AppCompatActivity() {
```

Comment:
since we are using single activity, I think we don't need to create `BaseActivity` anymore, how do you think? 😅

Reply:
That's actually a good point you're making, lol

Reply:
@manh-t @Tuubz hmm, I think we should keep defining base activity. That makes us focus on the logic of main activity when working on it, won't be messy with boilerplate code to build up something else.

Reply:
Sounds fair to me 🙏

Reply:
it's okay if you guys wanna keep it 🙆‍♂️